Source/Scene/Model.js

```diff
@@ -288,6 +288,7 @@ define([
      * @param {Color} [options.silhouetteColor=Color.RED] The silhouette color. If more than 256 models have silhouettes enabled, there is a small chance that overlapping models will have minor artifacts.
      * @param {Number} [options.silhouetteSize=0.0] The size of the silhouette in pixels.
      * @param {ClippingPlaneCollection} [options.clippingPlanes] The {@link ClippingPlaneCollection} used to selectively disable rendering the model.
+     * @param {String[]} [options.dequantizeInShader] The list of {@link https://github.com/KhronosGroup/glTF/blob/master/extensions/2.0/Khronos/KHR_draco_mesh_compression/README.md|KHR_draco_mesh_compression} encoded attributes to dequantize in the shader.
```

Comment:
We can only skip attributes by attribute type (https://github.com/google/draco/blob/master/src/draco/javascript/emscripten/draco_web_decoder.idl#L22), not each attribute. I think it may be best to limit to `POSITION`, `NORMAL`,
`COLOR`, and `TEX_COORD`.

(Three.js [only skips quantization for position](https://github.com/donmccurdy/three.js/blob/dev/examples/js/loaders/draco/DRACOLoader.js#L173))

Reply:
Ok, hardcoding those to be quantized sounds fine.